lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -54,23 +54,13 @@ class _ZoomableImageState extends State<ZoomableImage>
   bool _loadedLargeThumbnail = false;
   bool _loadingFinalImage = false;
   bool _loadedFinalImage = false;
-  ValueChanged<PhotoViewScaleState>? _scaleStateChangedCallback;
-  bool _isZooming = false;
   PhotoViewController _photoViewController = PhotoViewController();
 
   @override
   void initState() {
     _photo = widget.photo;
     _logger = Logger("ZoomableImage");
     _logger.info('initState for ${_photo.generatedID} with tag ${_photo.tag}');
-    _scaleStateChangedCallback = (value) {
-      if (widget.shouldDisableScroll != null) {
-        widget.shouldDisableScroll!(value != PhotoViewScaleState.initial);
```

Comment:
Do we then need `shouldDisableScroll`?

Reply:
no actually

Reply:
Then shouldn't we remove it from the Widget as well? And from whoever is passing it down here?